ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -183,6 +206,19 @@ internal constructor(
     }
   }
 
+  suspend fun updateDataSharingTerms(dataSharingTerms: Boolean) {
```

Comment:
Please rename this and related funs and vars to differentiate between the terms and the grant/consent.